saleor/graphql/product/mutations/products.py

```diff
@@ -1055,6 +1055,7 @@ def save(cls, info, instance, cleaned_input):
         attributes = cleaned_input.get("attributes")
         if attributes:
             AttributeAssignmentMixin.save(instance, attributes)
+        info.context.plugins.product_updated(instance)
```

Comment:
Is `order_updated` called when order is created? If yes, I assume we should have the same behaviour.
What about an update in the product's variants? Do we want to also call the hook in that case? cc @maarcingebala

Reply:
What about the situation when the product has been deleted? What about the mutation for Product META?

Reply:
> Is `order_updated` called when order is created? If yes, I assume we should have the same behaviour.

It's not called - there is a dedicated webhook product created for that, so I figured we shouldn't call update there.

> What about an update in the product's variants? Do we want to also call the hook in that case? cc @maarcingebala 

Added that according to the ticket, it makes sense.

About deleting the product - I think there should be separate webhook for that - product deleted.

About the META - probably we should trigger the update - it's an update to a product after all.

So the plan would be...
- create a ticket for product deleted webhook
- trigger product updated when META changes

CC @maarcingebala